### PR TITLE
Add updating existing rules section

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/contributing/contributing-rules.md [hunk 1]
```diff
@@ -493,4 +493,15 @@ References help provide more context to a developer on what the issue is, and ho
...
     ```
 
+## Updating existing rules
+
+To update an existing open-source rule, follow these steps:
+
+1. Find a rule you want to update in the [semgrep-rules](https://github.com/returntocorp/semgrep-rules/) repository.
+2. Submit a PR to the repository with your new update.
+3. Follow the same instructions and recommendations as you can find in the rest of this document. For example the security category has specific metadata requirements.
+4. Leave a message in the PR. Include what have you changed and why. What is the motivation for your update? TODO - add an example of a nicely written PR.
```
Comment (adamkvitek) [inline docs/contributing/contributing-rules.md:503]: @inkz @LewisArdern do you have an example of a great PR to our repository with a great message? We can share an old merged PR example here.

Comment (colleend) [reply docs/contributing/contributing-rules.md:503]: This looks good to me otherwise!

Comment (inkz) [reply docs/contributing/contributing-rules.md:503]: same! lgtm :+1:

**Review by inkz — APPROVED** [review]

File: docs/contributing/contributing-rules.md (hunk 1 above)
Comment (adamkvitek) [reply docs/contributing/contributing-rules.md:503]:
```suggestion
4. Leave a message in the PR. Explain why are you making changes. What is the motivation for this rule update?
```

**Review by s-santillan — APPROVED** [review]
lgtm!

**Pull request merged**